### PR TITLE
[14.0][ADD]attribut_set: manage company_dependent attributes

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

**Comment by xavier-bouquiaux** [comment]
LGTM <br/>
Used in production and work as designed <br/>

**Review by xavier-bouquiaux — COMMENTED** [review]
LGTM

File: attribute_set/models/ir_model.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class IrModelFields(models.Model):
+    _inherit = "ir.model.fields"
+
+    @api.model
+    def _instanciate_attrs(self, field_data):
+        attrs = super(IrModelFields, self)._instanciate_attrs(field_data)
```
Comment (rousseldenis) [inline attribute_set/models/ir_model.py]:
```suggestion
        attrs = super()._instanciate_attrs(field_data)
```

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Review by FrancoMaxime — APPROVED** [review]
I agree with @rousseldenis  's comment

File: attribute_set/models/ir_model.py (hunk 1 above)
Comment (xavier-bouquiaux) [reply attribute_set/models/ir_model.py]: Done and rebased

**Comment by hparfr** [comment]
Please review this pr:  https://github.com/OCA/odoo-pim/pull/138
It adds a migration script for the new field.

**Pull request merged**